subiquity/curtin.py

```diff
@@ -71,6 +71,11 @@
 # Reworked for flake8, but it does make it harder to read.
 POST_INSTALL_LIST = [
     ("late_commands:"),
+    ("    05_clear_subiquity: curtin in-target -- "
```

Comment:
Wait, so curtin works by copying it's own / to the point that snaps get transferred? That's a surprise to me.

Should we also remove the core snap?